host/xtest/Makefile

```diff
@@ -10,6 +10,10 @@ endif
 # be used instead.
 OPTEE_CLIENT_EXPORT ?= ../../../optee_client/out/export
 
+# If CFG_XTEST_LITE=n : compile full xtest suite (default)
+# If CFG_XTEST_LITE=y : compile sanity xtest suite
+CFG_XTEST_LITE ?= n
```

Comment:
+1 with @jenswi-linaro 
But this is not just a `if...` as the current `ifndef` is in the definition of the testcases to take into account. One solution is to add a boolen in `XTEST_AC_DSA_CASE` to indicate if the test case should be run in the light suite, and then add the `if` (same for #24)

So I would vote to have it in another PR (soon hopefully?) and to remove `CFG_XTEST_LITE` from the current PR